### PR TITLE
Remove k command and use dnx instead

### PR DESCRIPTION
**Pull request by ChengTian**
https://github.com/aspnet/dnx/issues/1473

@davidfowl @muratg

### CODE REVIEW

**Comment by ChengTian** [comment]
@davidfowl , this one should be checked in after CoreBuild get updated. After this one is checked in, I can check in all related PRs in all repos.

**Comment by ChengTian** [comment]
Already did local verification with updated KoreBuild

File: test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+            using (var runtimeHomeDir = TestUtils.GetRuntimeHomeDir(flavor, os, architecture))
+            {
+                string stdOut, stdErr;
+                var exitCode = BootstrapperTestUtils.ExecBootstrapper(
+                    runtimeHomeDir,
+                    arguments: ".",
+                    stdOut: out stdOut,
+                    stdErr: out stdErr);
+
+                Assert.NotEqual(0, exitCode);
```
Comment (troydai) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: Can you print the stdOut and stdErr if exit code is not zero?

File: test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+        {
+            var runtimeHomeDir = TestUtils.GetRuntimeHomeDir(flavor, os, architecture);
+            var projectStructure = @"{
+  'project.json': '{ }'
+}";
+
+            using (runtimeHomeDir)
+            using (var projectPath = TestUtils.CreateTempDir())
+            {
+
```
Comment (troydai) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: nit: remove this empty line.

File: test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs (hunk 1 above)
Comment (ChengTian) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: Not an issue for this thread

Comment (troydai) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: This comment doesn't stop you from merge this change.

Comment (ChengTian) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: Thanks, opened an issue: https://github.com/aspnet/dnx/issues/1496

File: test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs (hunk 2 above)
Comment (ChengTian) [inline test/Microsoft.Framework.ApplicationHost.FunctionalTests/AppHostTests.cs]: Fixed

**Comment by troydai** [comment]
:shipit:

**Comment by muratg** [comment]
:shipit:

**Pull request merged**